### ISSUE TITLE
Crash report from Dynamo 2.19.0.4737

### ISSUE THREAD

**Issue by aliasguru** [DynamoDS/Dynamo#15081]
# Issue Description
When I compile Dynamo myself, I cannot add Units node to a node tree. Crash happens immediately. I must be missing something on my end, as downloaded versions work fine on my system.

## Dynamo version
Dynamo: `2.19.0.4737`

## Operating system
OS: `Microsoft Windows NT 6.2.9200.0`

## What did you do?
Compile myself, run, new node tree, add `Units` node, bang.

## What did you expect to see?
a Units node obviously.

## What did you see instead?
crash report window

## What packages or external references (if any) were used?
No loaded packages were found.

## Stack Trace
```
The type initializer for 'DynamoUnits.Utilities' threw an exception.

   at DynamoUnits.Utilities.get_SchemaDirectory()
   at UnitsUI.Units.PopulateItemsCore(String currentSelection) in C:\DATEN\git-repos\Dynamo\src\Libraries\UnitsNodeModels\UnitNodeModels.cs:line 667
   at CoreNodeModels.DSDropDownBase.PopulateItems() in C:\DATEN\git-repos\Dynamo\src\Libraries\CoreNodeModels\DropDown.cs:line 303
   at CoreNodeModels.DSDropDownBase..ctor(String outputName) in C:\DATEN\git-repos\Dynamo\src\Libraries\CoreNodeModels\DropDown.cs:line 154
   at lambda_method(Closure )
   at Dynamo.Wpf.ViewModels.NodeSearchElementViewModel.OnClicked() in C:\DATEN\git-repos\Dynamo\src\DynamoCoreWpf\ViewModels\Search\NodeSearchElementViewModel.cs:line 370
   at Dynamo.UI.Controls.InCanvasSearchControl.ExecuteSearchElement(ListBoxItem listBoxItem) in C:\DATEN\git-repos\Dynamo\src\DynamoCoreWpf\Controls\InCanvasSearchControl.xaml.cs:line 72
   at Dynamo.UI.Controls.InCanvasSearchControl.OnMouseLeftButtonUp(Object sender, MouseButtonEventArgs e) in C:\DATEN\git-repos\Dynamo\src\DynamoCoreWpf\Controls\InCanvasSearchControl.xaml.cs:line 65
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.ReRaiseEventAs(DependencyObject sender, RoutedEventArgs args, RoutedEvent newEvent)
   at System.Windows.UIElement.OnPreviewMouseUpThunk(Object sender, MouseButtonEventArgs e)
   at System.Windows.RoutedEventArgs.InvokeHandler(Delegate handler, Object target)
   at System.Windows.RoutedEventHandlerInfo.InvokeHandler(Object target, RoutedEventArgs routedEventArgs)
   at System.Windows.EventRoute.InvokeHandlersImpl(Object source, RoutedEventArgs args, Boolean reRaised)
   at System.Windows.UIElement.RaiseEventImpl(DependencyObject sender, RoutedEventArgs args)
   at System.Windows.UIElement.RaiseTrustedEvent(RoutedEventArgs args)
   at System.Windows.Input.InputManager.ProcessStagingArea()
   at System.Windows.Input.InputManager.ProcessInput(InputEventArgs input)
   at System.Windows.Input.InputProviderSite.ReportInput(InputReport inputReport)
   at System.Windows.Interop.HwndMouseInputProvider.ReportInput(IntPtr hwnd, InputMode mode, Int32 timestamp, RawMouseActions actions, Int32 x, Int32 y, Int32 wheel)
   at System.Windows.Interop.HwndMouseInputProvider.FilterMessage(IntPtr hwnd, WindowMessage msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at System.Windows.Interop.HwndSource.InputFilterMessage(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndWrapper.WndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam, Boolean& handled)
   at MS.Win32.HwndSubclass.DispatcherCallbackOperation(Object o)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
   at System.Windows.Threading.Dispatcher.LegacyInvokeImpl(DispatcherPriority priority, TimeSpan timeout, Delegate method, Object args, Int32 numArgs)
   at MS.Win32.HwndSubclass.SubclassWndProc(IntPtr hwnd, Int32 msg, IntPtr wParam, IntPtr lParam)
   at MS.Win32.UnsafeNativeMethods.DispatchMessage(MSG& msg)
   at System.Windows.Threading.Dispatcher.PushFrameImpl(DispatcherFrame frame)
   at System.Windows.Application.RunDispatcher(Object ignore)
   at System.Windows.Application.RunInternal(Window window)
   at DynamoSandbox.DynamoCoreSetup.RunApplication(Application app) in C:\DATEN\git-repos\Dynamo\src\DynamoSandbox\DynamoCoreSetup.cs:line 73

-------- InnerException --------

Could not load file or assembly 'ForgeUnitsCLR.dll' or one of its dependencies. Das angegebene Modul wurde nicht gefunden.

   at DynamoUnits.Utilities.Initialize()
   at DynamoUnits.Utilities..cctor() in C:\DATEN\git-repos\Dynamo\src\Libraries\DynamoUnits\Utilities.cs:line 32

-------- Assemblies --------

mscorlib, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|DynamoSandbox, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|DynamoCore, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|PresentationFramework, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35|WindowsBase, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35|System.Core, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|System, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|PresentationCore, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35|System.Xaml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|DynamoInstallDetective, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|System.Configuration, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|DynamoApplications, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|DynamoServices, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|netstandard, Version=2.0.0.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51|CommandLine, Version=2.8.0.0, Culture=neutral, PublicKeyToken=5a870481e358d379|DynamoUtilities, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|DynamoCore.resources, Version=2.19.0.4737, Culture=en-US, PublicKeyToken=null|System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|DynamoCoreWpf, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|Microsoft.Practices.Prism, Version=4.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35|Microsoft.Web.WebView2.Core, Version=1.0.1264.42, Culture=neutral, PublicKeyToken=2a8ab48044d2601e|Microsoft.Web.WebView2.Wpf, Version=1.0.1264.42, Culture=neutral, PublicKeyToken=2a8ab48044d2601e|DynamoCoreWpf.resources, Version=2.19.0.4737, Culture=en-US, PublicKeyToken=null|PresentationFramework.Aero2, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35|mscorlib.resources, Version=4.0.0.0, Culture=de, PublicKeyToken=b77a5c561934e089|System.Drawing, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|Microsoft.VisualStudio.DesignTools.WpfTap, Version=17.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|System.Runtime.Serialization, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|SMDiagnostics, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|System.ServiceModel.Internals, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35|System.Runtime.Serialization.resources, Version=4.0.0.0, Culture=de, PublicKeyToken=b77a5c561934e089|UIAutomationProvider, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35|UIAutomationTypes, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35|System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|SharpDX.Mathematics, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b4dcf0f35e5521f1|Microsoft.GeneratedCode, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null|DynamoApplications.resources, Version=2.19.0.4737, Culture=en-US, PublicKeyToken=null|DynamoShapeManager, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|Greg, Version=2.5.0.5076, Culture=neutral, PublicKeyToken=null|RestSharp, Version=106.12.0.0, Culture=neutral, PublicKeyToken=598062e77f915f75|ProtoCore, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|ProtoAssociative, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|ProtoImperative, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|Lucene.Net, Version=4.0.0.0, Culture=neutral, PublicKeyToken=85089178b9ac3181|Lucene.Net.QueryParser, Version=4.0.0.0, Culture=neutral, PublicKeyToken=85089178b9ac3181|Analytics.NET.ADP, Version=4.0.0.1432, Culture=neutral, PublicKeyToken=null|Analytics.NET.Core, Version=4.0.0.1432, Culture=neutral, PublicKeyToken=null|AdpSDKCSharpWrapper, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null|Microsoft.CSharp, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|Anonymously Hosted DynamicMethods Assembly, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null|System.Dynamic, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|System.Web.Extensions, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35|System.Web, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|Newtonsoft.Json, Version=13.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed|DynamoPackages, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|DesignScriptBuiltin, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|VMDataBridge, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|ProtoCore.resources, Version=2.19.0.4737, Culture=en-US, PublicKeyToken=null|ProtoGeometry, Version=2.18.0.2485, Culture=neutral, PublicKeyToken=null|LibG.Interface, Version=2.18.0.2485, Culture=neutral, PublicKeyToken=null|ProtoGeometry.resources, Version=2.18.0.2485, Culture=en-US, PublicKeyToken=null|DSCoreNodes, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|DSCoreNodes.resources, Version=2.19.0.4737, Culture=en-US, PublicKeyToken=null|DSOffice, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|DSOffice.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|DSCPython, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|Python.Runtime, Version=2.5.2.7195, Culture=neutral, PublicKeyToken=null|DynamoConversions, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|DynamoUnits, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|System.Numerics, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|System.Data, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|Tessellation, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|MIConvexHull.NET Standard, Version=1.0.17.411, Culture=neutral, PublicKeyToken=null|System.Runtime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|Analysis, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|Analysis.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|GeometryColor, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|J2N, Version=2.0.0.0, Culture=neutral, PublicKeyToken=f39447d697a969af|System.Runtime.InteropServices.RuntimeInformation, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|Microsoft.Extensions.Configuration.Abstractions, Version=1.1.2.0, Culture=neutral, PublicKeyToken=adb9793829ddae60|Microsoft.Extensions.Primitives, Version=1.1.1.0, Culture=neutral, PublicKeyToken=adb9793829ddae60|Lucene.Net.Analysis.Common, Version=4.0.0.0, Culture=neutral, PublicKeyToken=85089178b9ac3181|ProtoScript, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|Autodesk.IDSDK, Version=1.1.6.0, Culture=neutral, PublicKeyToken=null|CoreNodeModels.customization, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|CoreNodeModels, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|CoreNodeModels.resources, Version=2.19.0.4737, Culture=en-US, PublicKeyToken=null|CoreNodeModelsWpf.customization, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|CoreNodeModelsWpf, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|CoreNodeModelsWpf.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|GeometryUI.customization, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|GeometryUI, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|GeometryUI.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|GeometryUIWpf, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|Migrations, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|NCalc, Version=1.3.8.0, Culture=neutral, PublicKeyToken=973cde3f1cafed03|PythonNodeModels.customization, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|PythonNodeModels, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|PythonNodeModels.resources, Version=2.19.0.4737, Culture=en-US, PublicKeyToken=null|PythonNodeModelsWpf, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|ICSharpCode.AvalonEdit, Version=6.3.0.90, Culture=neutral, PublicKeyToken=9cc39be672370310|TestUINodes, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|UnitsNodeModels, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|UnitsNodeModels.customization, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|UnitsNodeModels.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|UnitsUI.customization, Version=2.19.0.5586, Culture=neutral, PublicKeyToken=null|UnitsUI, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|Watch3DNodeModels.customization, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|Watch3DNodeModels, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|Watch3DNodeModels.resources, Version=2.19.0.4737, Culture=en-US, PublicKeyToken=null|Watch3DNodeModelsWpf, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|System.Xml.Linq, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|BuiltIn.customization, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|Operators.customization, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|ProtoGeometry.customization, Version=2.18.0.2485, Culture=neutral, PublicKeyToken=null|DesignScriptBuiltin.customization, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|DSCoreNodes.customization, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|DSOffice.customization, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|DynamoUnits.customization, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null|Tessellation.customization, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|Analysis.customization, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|GeometryColor.customization, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|KiskaPatternNodes, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null|DynamoPackages.resources, Version=2.19.0.4737, Culture=en-US, PublicKeyToken=null|HelixToolkit.Wpf.SharpDX, Version=2.17.0.0, Culture=neutral, PublicKeyToken=52aa3500039caf0d|SharpDX, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b4dcf0f35e5521f1|HelixToolkit, Version=2.17.0.0, Culture=neutral, PublicKeyToken=52aa3500039caf0d|SharpDX.DXGI, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b4dcf0f35e5521f1|SharpDX.Direct3D11, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b4dcf0f35e5521f1|SharpDX.Direct2D1, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b4dcf0f35e5521f1|System.Collections.Immutable, Version=5.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|FontAwesome.WPF, Version=4.7.0.37774, Culture=neutral, PublicKeyToken=0758b07a11a4f466|System.Resources.ResourceManager, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|PresentationFramework-SystemCore, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|PresentationFramework-SystemXml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|DocumentationBrowserViewExtension, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|DocumentationBrowserViewExtension.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|DynamoManipulation, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|GraphMetadataViewExtension, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|GraphNodeManagerViewExtension, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|LibraryViewExtensionWebView2, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|LintingViewExtension, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|Notifications, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|PackageDetailsViewExtension, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|DynamoPackagesWPF, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|PythonMigrationViewExtension, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|WorkspaceDependencyViewExtension, Version=2.19.4.8225, Culture=neutral, PublicKeyToken=null|WorkspaceDependencyViewExtension.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|GraphMetadataViewExtension.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|GraphNodeManagerViewExtension.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|LintingViewExtension.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|Notifications.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|PackageDetailsViewExtension.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|PresentationFramework-SystemXmlLinq, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|Dynamo.Microsoft.Xaml.Behaviors, Version=1.1.0.0, Culture=neutral, PublicKeyToken=null|PresentationFramework-SystemData, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|DotNetProjects.Wpf.Extended.Toolkit, Version=5.0.103.0, Culture=neutral, PublicKeyToken=3e4669d2f30244f4|LibraryViewExtensionWebView2.resources, Version=2.19.4.8225, Culture=en-US, PublicKeyToken=null|Cyotek.Drawing.BitmapFont, Version=1.0.0.0, Culture=neutral, PublicKeyToken=58daa28b0b2de221|WindowsFormsIntegration, Version=4.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35|SharpDX.D3DCompiler, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b4dcf0f35e5521f1|SharpDX.Direct3D9, Version=4.2.0.0, Culture=neutral, PublicKeyToken=b4dcf0f35e5521f1|Accessibility, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|Windows.UI, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null, ContentType=WindowsRuntime|Windows.Foundation, Version=255.255.255.255, Culture=neutral, PublicKeyToken=null, ContentType=WindowsRuntime|System.Runtime.InteropServices.WindowsRuntime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a|System.Runtime.WindowsRuntime, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089|Prism, Version=7.2.0.1422, Culture=neutral, PublicKeyToken=40ee6c3a2184dc59|DynamoCore.customization, Version=2.19.0.4737, Culture=neutral, PublicKeyToken=null
```

---
CLR: `4.0.30319.42000`

**Comment by aliasguru** [comment]
Same problem, same solution as with https://github.com/DynamoDS/Dynamo/issues/13826 - but this time I can't solve it. Going back to 16.1 makes Units work again, but then switching back to 19.0 breaks them again. Stuck in an endless loop at the moment.

Can anyone please try checking out the Dynamo repo from scratch with one of the latest branches (19.0 onwards) and try compilation, run and add a Units node?

**Comment by aliasguru** [comment]
Came closer to the problem. The Units node mentioned only crashes when I try to add it using the right click menu. When I go through the interface, clicking on the items marked with an `X` in the image below simply does nothing:

![DynamoUnitsNodes](https://github.com/DynamoDS/Dynamo/assets/24734924/829e8467-ec29-44e5-84a2-ac4e9af8541d)

The Units node I am using is one of them.

**Comment by mjkkirschner** [comment]
cherry pick this PR to your branch:
https://github.com/DynamoDS/Dynamo/pull/15031/files

**Comment by mjkkirschner** [comment]
your other solution probably worked because you built a version that copied the file, then built a version that did not copy it so it was still in your bin folder.

**Comment by aliasguru** [comment]
> cherry pick this PR to your branch: https://github.com/DynamoDS/Dynamo/pull/15031/files
> 
> or build 2.19.5

thanks, cherry picking solved the issue! Can't use 19.5 as the ProtoGeometry library does not work with my Alias 2024.1 installation (am using the -gp flag, still no luck), but that's a different topic.
